src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/grid.html

```diff
@@ -1,6 +1,6 @@
 <div ng-controller="Umbraco.PropertyEditors.GridController" class="umb-grid umb-property-editor clearfix" id="umb-grid">
 
-    <umb-editor-sub-header ng-if="showReorderButton()" style="background-color: white;">
+    <umb-editor-sub-header ng-if="showReorderButton()" style="background-color: white; border-color: white;">
```

Comment:
Any good reason to add more inline styles here?

Reply:
As you wish, it is here by done.. :-D